internal/node/controller/auth.go

```diff
@@ -103,3 +109,186 @@ func getUserCertAndInfo(controller NodeController, username string) (*x509.Certi
 
 	return cert, user, nil
 }
+
```

Comment:
No reason this needs to be in controller actually ... could just be in `internal/bff`

Reply:
I made this change